frontend/i18n-scripts/memsource-download.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+set -exuo pipefail
+
+source ./i18n-scripts/languages.sh
+
+# Before running this tool, you must first install the unofficial Memsource CLI client: https://github.com/unofficial-memsource/memsource-cli-client#pip-install
+# You also have to configure it with your login info: https://github.com/unofficial-memsource/memsource-cli-client#configuration-red-hat-enterprise-linux-derivatives
+
+# On MacOS, you must also install rename, a GNU command line utility: brew install rename
+
+# Example CLI usage for this script: yarn memsource-download -p 6sB6qwpbRkGCeBQq4hUyK1
+# The project ID can be obtained from the Memsource project URL
+# (the series of numbers and letters after /show/)
+# i.e. https://cloud.memsource.com/web/project2/show/FBfZeTEWPYaC4VXhgrW0R2
+
+while getopts p: flag
+do
+    case "${flag}" in
+        p) PROJECT_ID=${OPTARG};;
+        *) echo "usage: $0 [-p]" >&2
+        exit 1;;
+    esac
+done
+
+echo "Downloading PO files from Project ID \"$PROJECT_ID\""
+
+# Memsource job listing is limited to 50 jobs per page
+# We need to pull all the files down by page and stop when we reach a page with no data
+for i in "${LANGUAGES[@]}"
+do
+  COUNTER=0
+  CURRENT_PAGE=$(memsource job list --project-id $PROJECT_ID --target-lang $i -f value --page-number 0 -c uid | tr '\n' ' ')
```

Comment:
I'd add quotes where you expand variables here and below.

```suggestion
  CURRENT_PAGE=$(memsource job list --project-id "$PROJECT_ID" --target-lang "$i" -f value --page-number 0 -c uid | tr '\n' ' ')
```